### PR TITLE
Use VA Profile v3 service to obtain NOK/EC

### PR DESCRIPTION
**Pull request by radavis**
## Summary
- `GET /v0/profile/contacts` retrieves contacts from the VA Profile v3 service

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#67088
- Will replace work done in department-of-veterans-affairs/vets-api#14024, as we've been instructed to use the VA Profile v3 service, and not the Health Benefit service.

## Testing done

- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
- *Describe the tests completed and the results*

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Comment by radavis** [comment]
Note: The @department-of-veterans-affairs/disability-experience team was added as a reviewer due to the following line in `.github/CODEOWNERS`: 

```
lib/va_profile @department-of-veterans-affairs/Disability-Experience @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

**Review by tpharrison — APPROVED** [review]
LGTM 🚀

**Comment by RachalCassity** [comment]
@radavis Can you please pull from master to clear out the codeowner errors?

**Comment by radavis** [comment]
@RachalCassity, pulling master cleared previous codeowners errors. Currently, waiting on [this request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/74002) for giving the `vfs-mhv-integration` team `vets-api` write access, which should clear the rest of the errors with this file.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1942,16 +1943,19 @@ spec/uploaders/validate_pdf_spec.rb @department-of-veterans-affairs/Disability-E
...
-spec/controllers/v0/profile/contacts_controller_spec.rb @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/vfs-health-modernization-initiative @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-spec/fixtures/va_profile/health_benefit_v1_associated_persons.json @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/vfs-health-modernization-initiative @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-spec/lib/va_profile/health_benefit/service_spec.rb @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/vfs-health-modernization-initiative @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+app/controllers/v0/profile/contacts_controller.rb @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/vfs-mhv-integration @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```
Comment (RachalCassity) [inline .github/CODEOWNERS:1946]: Github is flagging @department-of-veterans-affairs/vfs-mhv-integration as an unknown owner

Comment (radavis) [reply .github/CODEOWNERS:1946]: This is the link to the team. Does it need to be made public? https://github.com/orgs/department-of-veterans-affairs/teams/vfs-mhv-integration/members

Comment (radavis) [reply .github/CODEOWNERS:1946]: And/or does the vets-api repo need to be added to this page? https://github.com/orgs/department-of-veterans-affairs/teams/vfs-mhv-integration/repositories

**Review by ericboehs — APPROVED** [review]
I checked with Rachal and this PR is good to go.

**Pull request merged**